### PR TITLE
Changed: code samples to support changes in api.py

### PR DESCRIPTION
**Pull request by iftwigs**

### CODE REVIEW

File: docs/sdk/tutorials/regex_based_annotations/index_nb.md [hunk 1]
```diff
@@ -43,14 +43,17 @@ In this guide, we'll show you how to use Python and regular expressions (regex)
 Let's say we have a Document, and we want to highlight every instance of the term "Musterstraße", which might represent a specific street name or location. Our task is to find this term, label it as "Lohnart", and associate it with the "Brutto-Bezug" Label Set.
 
 ```python editable=true slideshow={"slide_type": ""} tags=["remove-cell"] vscode={"languageId": "plaintext"}
+import sys
+sys.path.insert(0, '../../../../')
```
Comment (zypriafl) [inline docs/sdk/tutorials/regex_based_annotations/index_nb.md:47]: Why is this needed now?  Please add a comment.

**Review by zypriafl — CHANGES_REQUESTED** [review]

File: docs/sdk/tutorials/regex_based_annotations/index_nb.md (hunk 1 above)
Comment (iftwigs) [reply docs/sdk/tutorials/regex_based_annotations/index_nb.md:48]: this is needed for importing variables from "tests". added the comments everywhere where this was changed.
